docs/changelog.md

```diff
@@ -1,6 +1,7 @@
 dev
 
 - Fixed `pipx list` output phrasing to convey that python version displayed is the one with which package was installed. 
+- Fixed `pipx install` to provide return code 0 if venv already exists, similar behavior as pip. (#736)
```

Comment:
```suggestion
- Fixed `pipx install` to provide return code 0 if venv already exists, similarly to pip’s behavior. (#736)
```

Reply:
Thanks for the suggestion, updated and new commit pushed accordingly.